src/backend/TrafficCourts/Messaging/MessageContracts/SubmitNoticeOfDispute.cs

```diff
@@ -210,5 +210,10 @@ public class SubmitNoticeOfDispute : IMessage
     /// </summary>
     public string? OcrViolationTicket { get; set; }
 
+    /// <summary>
+    /// Origin of request for sending email verification
+    /// </summary>
+    public string Host { get; set; } = String.Empty;

```

Comment:
Does the "Host" really need to be a part of "SubmitNoticeOfDispute" message or can it be handled in a different message since it's for passing the details of a Notice Of Dispute?

Reply:
its needed for the email address verification but may be removed as part of work in a future PR